### DISCUSSION TITLE
编译固件默认带上serverchan或pushbot

### DISCUSSION THREAD

**Discussion by fulracoco** [coolsnowwolf/lede#9639] [R68S / 电犀牛专区]
rt,网络变化有通知。

**Comment by Plutonium141** [comment]
这个你可以自己安装的
[luci-app-pushbot](https://op.dllkids.xyz/packages/aarch64_generic/luci-app-pushbot_git-22.133.34391-1952137_all.ipk)

**Reply by fulracoco** [reply]
> 是的，但是升级之后会丢失，要手动重新安装一遍

**Reply by Plutonium141** [reply]
那这个没有办法 等开源后自行编译吧
